src/Akka.HealthCheck.IHealthCheckExample/Program.cs

```diff
@@ -0,0 +1,54 @@
+using Akka.Cluster.Hosting;
+using Akka.HealthCheck.Cluster;
+using Akka.HealthCheck.IHealthCheckExample;
+using Akka.HealthCheck.IHealthCheckExample.HealthChecks;
+using Akka.HealthCheck.Liveness;
+using Akka.HealthCheck.Persistence;
+using Akka.HealthCheck.Readiness;
+using Akka.Hosting;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddAkka("actor-system", (configurationBuilder, serviceProvider) =>
+{
+    configurationBuilder
+    .WithClustering()
+    .WithActors((actorSystem, registry) =>
+    {
+        // We need to start the Probe actors which we need in our HealthChecks
+        actorSystem.CreateAndRegisterHealthProbe<DefaultLivenessProbe>(registry, new DefaultLivenessProvider(actorSystem));
+        actorSystem.CreateAndRegisterHealthProbe<ClusterLivenessProbe>(registry, new ClusterLivenessProbeProvider(actorSystem));
+        actorSystem.CreateAndRegisterHealthProbe<AkkaPersistenceLivenessProbe>(registry, new AkkaPersistenceLivenessProbeProvider(actorSystem));
```

Comment:
This persistence liveness check actor needs to be singleton in a cluster. The persistenceId will otherwise try to write data from all nodes under the same persistenceId to the same storage. That will cause conflicts resulting in the fact that your cluster will become unstable now and then (cc: @Aaronontheweb)